gradle.properties

```diff
@@ -1,4 +1,21 @@
 org.gradle.caching=true
 group=org.ballerinalang.sf
 version=2.2.1
-ballerinaLangVersion=2.0.0-beta.2.1
+ballerinaLangVersion=2.0.0-beta.2
```

Comment:
`2.0.0-beta.2.1` is the latest and correct version for ballerinaLangVersion.
Don't change this to `2.0.0-beta.2`.

But for the other stdlib versions, u can use beta.2
Ex: `stdlibIoVersion=0.6.0-beta.2`

Reply:
I have used a dependency called jballerina-tools to build the ballerina project during the gradle build. When I try to import it in 2.0.0-beta2.1 version it is not available. So as to be consistent with the ballerina versions I have used this lang version. Have to check if some version related problems occur when using different versions of ballerina lang and jballerina tools.